### ISSUE TITLE
With Chrome extension, download does not start with Motrix despite the shared key, how to fix ?

### ISSUE THREAD

**Issue by oeloo** [gautamkrishnar/motrix-webextension#140]
Hi, I have the exact same key and port set in Motrix for Windows (v1.8.19) and Motrix Chrome extension (v23.12.10) but when I download a file (for example a .rar) from Chrome, Motrix is starting to download it, it is the Chrome built-in download system that is downloading the file. How to fix this ?
![image](https://github.com/gautamkrishnar/motrix-webextension/assets/98819304/fbcba25e-977f-4ef8-99e3-05ef26409bc0)
![image](https://github.com/gautamkrishnar/motrix-webextension/assets/98819304/10f6f621-5848-4800-b684-08063898131f)

**Comment by redactedscribe** [comment]
I have the same issue but in Firefox. It's quite essential to have an extension to use else Motrix loses its value to me. I guess I'll stick with [HTTP Downloader](https://github.com/erickutcher/httpdownloader) (Windows only) for now.

**Comment by oeloo** [comment]
> It's quite essential to have an extension to use else Motrix loses its value to me.
I fully agree, the Motrix browser extension is essential. Why nobody cares about this issue? it is a major bug

**Comment by 0999ad** [comment]
I, too, am having this very same issue.
It was working, then stopped, I re-did all the configs as shown above.
still using the Chrome download, not Motrix.
